src/CloudEvents/Event.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CloudEvents;
+
+class Event
```

Comment:
I think the class name `CloudEvent` would make more sense. See the .NET SDK for an example.

Reply:
Think this is a good call — I'll make that change.

Seems there's a split in SDK standards for the name, but for context, I preferred Event because of the [notations and terminology](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#notations-and-terminology) section of the spec. A lot of the spec refers to `CloudEvent` as a more concrete entity compared to an event.

@rubenrangel and I talked about the idea of the Event class as an interface altogether, which might lean more towards the Event name.